dev/com.ibm.ws.ui.tool.serverConfig/resources/WEB-CONTENT/nls/messages.js

```diff
@@ -146,7 +156,12 @@ var editorMessages = {
 "AUTH_ALIAS_OPTIONAL": "Authentication alias (Optional)",
 "AUTH_ALIAS": "Authentication alias",
 "CONTAINER_AUTHENTICATION": "Container authentication",
+"RESOURCE_REFERENCE": "Resource reference",
 "NO_RESOURCE_REFERENCE": "No resource reference",
+"RESOURCE_REFERENCE_WITH_APPILCATION_AUTHENTICATION_IN_USE": "Test the connection by using a resource reference that authenticates with application authentication.",
+"RESOURCE_REFERENCE_WITH_CONTAINER_AUTHENTICATION_IN_USE": "Test the connection by using a resource reference that authenticates with container authentication.",
+"RESOURCE_REFERENCE_IN_USE": "Container authentication for direct lookups is enabled. The connection test will use the configured default container authentication data.",
```

Comment:
per IBM style, avoid future tense
The connection test will use the configured default container authentication data.
--->
The connection test uses the configured default container authentication data.